doc/source/algorithms.rst

```diff
@@ -184,11 +184,27 @@ The user can (optionally) provide a visitor that perform operations along with t
 
     Invoked when a vertex is closed (all its neighbors have been examined).
 
-.. py:function:: dijkstra_shortest_paths_explicit(graph, edge_dists, source[, visitor])
-Returns the explicit paths using ``dijkstra_shortest_paths()`` as an array of
-vectors of vertices from the source to each target. Empty vectors are used to
-indicate vertices that are unreachable from the source. Parameters are the same
-as described in ``dijkstra_shortest_paths()``.
+.. py:function:: enumerate_paths(result[, dest])
+
+    Returns an array of vectors ``all_paths`` (containing vertices), where ``all_paths[i]`` corresponds to the path from a source to vertex ``i``. Empty vectors are used to indicate vertices that are unreachable from the source. Additionally, ``isdefined(all_paths, i)==true`` for all vertices ``i`` that are in ``dest``, and ``false`` otherwise, where ``dest`` can be a single vertex, a subset of vertices, or left unspecified (in which case, all the vertices in the graph will be considered).
+
+    The following is an example that shows how to use this function:
+
+.. code-block:: python
+
+    julia> s3 = dijkstra_shortest_paths(g3,2)
+    julia> sps = enumerate_paths(s3)
+    4-element Array{Array{Int64,1},1}:
+     []
+     [2]
+     [2,3]
+     [2,3,4]
+    julia> sps = enumerate_paths(s3, [2,4])
```

Comment:
Do you think it will be a common use case to give the input as an unordered set instead of a simple vector? This input format isn't used anywhere else in the package.